flume/doc/quick_start_guide.md

```diff
@@ -1 +1,154 @@
+
+```
+$ cat > /etc/yum.repos.d/fiware.repo <<EOL
+[Fiware]
+name=FIWARE repository
+baseurl=http://repositories.testbed.fi-ware.eu/repo/rpm/x86_64/
+gpgcheck=0
+enabled=1
+EOL
+$ yum install cygnus
+```
+
+When installing through the RPM, Cygnus is started as a service. Stop it since this Quick Start Guide will run Cygnus from the command line.
+
+```
+$ service cygnus stop
+```
+
+##Configuring a test agent
+This kind of agent is the simplest one you can configure with Cygnus. It is based on a standard `HTTPSource`, a `MemoryChannel` and a `OrionTestSink`. Don't worry about the configuration details, specially those about the source; simply think on a Http listener waiting for Orion notifications on port TCP/5050 and sending that notifications in the form el Flume events to a testing purpose sink that will not really persist anything in a third-party storage, but will log the notified context data.
+
+(1) Instantiate an `agent_test.conf` file from the distributed templates:
+
+```
+$ cp /usr/cygnus/conf/agent.conf.template /usr/cygnus/conf/agent_test.conf
+```
+
+(2) Edit `/usr/cygnus/conf/agent_test.conf`:
+
+```
+cygnusagent.sources = http-source
+cygnusagent.sinks = test-sink
+cygnusagent.channels = test-channel
+
+cygnusagent.sources.http-source.channels = test-channel
+cygnusagent.sources.http-source.type = org.apache.flume.source.http.HTTPSource
+cygnusagent.sources.http-source.port = 5050
+cygnusagent.sources.http-source.handler = es.tid.fiware.fiwareconnectors.cygnus.handlers.OrionRestHandler
+cygnusagent.sources.http-source.handler.notification_target = /notify
+cygnusagent.sources.http-source.handler.default_service = def_serv
+cygnusagent.sources.http-source.handler.default_service_path = def_servpath
+cygnusagent.sources.http-source.handler.events_ttl = 2
+cygnusagent.sources.http-source.interceptors = ts de
+cygnusagent.sources.http-source.interceptors.ts.type = timestamp
+cygnusagent.sources.http-source.interceptors.de.type = es.tid.fiware.fiwareconnectors.cygnus.interceptors.DestinationExtractor$Builder
+cygnusagent.sources.http-source.interceptors.de.matching_table = /Applications/apache-flume-1.4.0-bin/conf/matching_table.conf
+
+cygnusagent.channels.test-channel.type = memory
+cygnusagent.channels.test-channel.capacity = 1000
+cygnusagent.channels.test-channel.transactionCapacity = 100
+
+cygnusagent.sinks.test-sink.channel = test-channel
+cygnusagent.sinks.test-sink.type = es.tid.fiware.fiwareconnectors.cygnus.sinks.OrionTestSink
+```
+
+(3) Start Cygnus from the command line; this will be printing logs on the standard output (i.e. your screen):
+
+```
+$ /usr/cygnus/bin/cygnus-flume-ng agent --conf /usr/cygnus/conf/ -f /usr/cygnus/conf/cygnus_test.conf -n cygnusagent -Dflume.root.logger=INFO,console
+```
+
+(4) Open a new terminal (since Cygnus is printing logs on the standard output of the first one) and edit somewhere a `notification-json-simple.sh` file:
```

Comment:
Regarding (3), probably stating that a JRE is needed and that JAVA_HOME should point to it is desirable to avoid getting an Exception :) Anyhow, in the provided output, this need is stated:
```
[german@frb-dev ~]$ /usr/cygnus/bin/cygnus-flume-ng agent --conf /usr/cygnus/conf/ -f /usr/cygnus/conf/cygnus_test.conf -n cygnusagent -Dflume.root.logger=INFO,console
Warning: JAVA_HOME is not set!
+ exec /usr/bin/java -Xmx20m -Dflume.root.logger=INFO,console -cp '/usr/cygnus/conf:/usr/cygnus/lib/*:/usr/cygnus/plugins.d/cygnus/lib/*:/usr/cygnus/plugins.d/cygnus/libext/*' -Djava.library.path= es.tid.fiware.fiwareconnectors.cygnus.nodes.CygnusApplication -f /usr/cygnus/conf/cygnus_test.conf -n cygnusagent
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/usr/cygnus/lib/slf4j-log4j12-1.6.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/usr/cygnus/plugins.d/cygnus/lib/cygnus-0.7.0_SNAPSHOT-jar-with-dependencies.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
Exception in thread "main" java.lang.NullPointerException
	at es.tid.fiware.fiwareconnectors.cygnus.nodes.CygnusApplication.main(CygnusApplication.java:181)
```